### PR TITLE
Update quest-modding.md

### PR DESCRIPTION
**Pull request by mrstephenson2142**
Added additional details to QuestPatcher instructions.

### CODE REVIEW

**Comment by mrstephenson2142** [comment]
I just saw that there was a guide on making submissions. I understand if this gets rejected due to not following a workflow.

File: wiki/quest-modding.md [hunk 1]
```diff
@@ -155,7 +155,17 @@ You will need to temporarily remove all secondary accounts before modding the ga
...
+  - Only perform the downgrade steps. Do not perform any steps related to BMBF installation or configuration. 
+
+:::
+
 1. Download and install [QuestPatcher](https://github.com/Lauriethefish/QuestPatcher/releases/latest)
 2. When patching, make sure `QuestLoader` is selected, or your mods will not load.
 3. Patch Beat Saber (or re-patch in the tools tab if you have already)
-4. Install the core mods, and other mods you would like from [this page.](https://computerelite.github.io/tools/Beat_Saber/questmods.html)
+4. Install the core mods, and other mods you would like from [this page.](https://computerelite.github.io/tools/Beat_Saber/questmods.html).
+    - It is recommended to install the "BetterSongSearch" mod since it provides more search features.
```
Comment (bloodcloak) [inline wiki/quest-modding.md:170]: would remove as prefer not call out specific mods for preferences like this

File: wiki/quest-modding.md [hunk 2]
```diff
@@ -155,7 +155,17 @@ You will need to temporarily remove all secondary accounts before modding the ga
...
+    - You will have to download the mods, then load them using QuestPatcher.
```
Comment (bloodcloak) [inline wiki/quest-modding.md:171]: would need clarification on downloading location, like on a computer or via headset?

File: wiki/quest-modding.md [hunk 3]
```diff
@@ -155,7 +155,17 @@ You will need to temporarily remove all secondary accounts before modding the ga
 You can add them back once the modding process is completed.
 :::
 
+:::tip NOTE
```
Comment (bloodcloak) [inline wiki/quest-modding.md:158]: probably better to copy the whole downgrade section to here instead of forwarding to the bmbf page

**Review by bloodcloak — CHANGES_REQUESTED** [review]

**Pull request closed without merging**